docs/notebooks/dmri_covariance.ipynb

```diff
@@ -76,6 +85,89 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.scatter(thetas_emp, b3000_cov, s=0.3, c=\"lightgrey\", label=\"Empirical\")\n",
+    "\n",
+    "# Plot the exponential and spherical model covariances\n",
+    "plt.plot(theta_lin, cov_exp * 2, \"gray\", label=\"Exponential\")\n",
+    "plt.plot(theta_lin, cov_sph * 2, \"k--\", label=\"Spherical\")\n",
```

Comment:
```suggestion
    "# Scale model by a factor of 2 so that it matches Andersson2015's fig. 3.\n",
    "plt.plot(theta_lin, cov_exp * 2, \"gray\", label=\"Exponential\")\n",
    "plt.plot(theta_lin, cov_sph * 2, \"k--\", label=\"Spherical\")\n",
```